defaults/main.yml

```diff
@@ -3,7 +3,7 @@ node_exporter_config_dir: /var/lib/node_exporter
 node_exporter_run_in_docker: false
 node_exporter_docker_network_name: prom_network
 node_exporter_container_name: node_exporter
-node_exporter_docker_image: "quay.io/prometheus/node-exporter:1.1.2"
+node_exporter_docker_image: "quay.io/prometheus/node-exporter:1.4.0"
```

Comment:
It won't work simply because version you point doesn't exists. And overall - you have to upgrade both docker and non-docker versions.